tox.ini

```diff
@@ -123,3 +123,5 @@ commands =
 deps =
     # If you need to test on a certain branch, add @<branch-name> after.git
     git+https://github.com/amazon-braket/amazon-braket-sdk-python.git
+    git+https://github.com/aws/amazon-braket-schemas-python.git@main 
+    git+https://github.com/aws/amazon-braket-default-simulator-python.git@main
```

Comment:
nit: doesn't affect functionally, should the org be `amazon-braket` to be consistent in the urls?

Reply:
As well as the explicit branch name